pkg/components/istio-operator/manifest.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package istiooperator has code related to deployment of istio operator component.
```

Comment:
Why not `doc.go` for documentation as we usually do for other packages?

Reply:
We don't do it for one line package comments. If it has significant amount of documentation then we can move it to a dedicated file.

Reply:
+1 for `doc.go`

Reply:
Added one.